release.wdl

```diff
@@ -0,0 +1,238 @@
+#Thoughts--
+#Still need to manually check that the jars are being created in broad artifactory
+#Need to manually investigate if tests aren't being written
+#Need to manualy update the release portion of Cromwell
+#Need to make sure that the changelog.md file doesn't get weirdly altered when merging to master for Cromwell
+
+
+task basicRelease {
+   Array[String] dependency
+   String repo
+   String releaseV
+   String baseDir
+
+   command {
+     cd ${baseDir}/${repo}
+     git checkout develop
```

Comment:
The basic assumption I made is that entire WDL will be run locally on our laptops, and so for each task, you just need to provide the path for where the repo's are on your machine.